Include/internal/pycore_list.h

```diff
@@ -37,6 +37,24 @@ struct _Py_list_state {
 
 #define _PyList_ITEMS(op) (_PyList_CAST(op)->ob_item)
 
+PyAPI_FUNC(int)
+_PyList_AppendTakeRefListResize(PyListObject *self, PyObject *newitem);
+
+static inline int
```

Comment:
No need for an inline function.
Let the LTO pass decide if it wants to inline it.